doc/source/user_guide/adaptive_templates.rst

```diff
@@ -261,17 +254,14 @@ Region properties can be set using the appropriate field/property:
 If the region object of the rotor has been created in Python (``rotor = mc.get_region("Rotor")``)
 The rotor region object's properties can be obtained and set for the rotor notch.
 
-The ``Region.duplications`` property represents the symmetry of the region.
-In the example shown using the e9 IPM template, ``duplications = 8``
-because there are 8 rotor poles of 45 ° symmetry.
+The ``Region.duplications`` property represents the symmetry of the region. In the example shown
+using the e9 IPM template, ``duplications = 8`` because there are 8 rotor poles of 45 ° symmetry.
 In this example, the notch would have the same symmetry as the rotor.
 
-The parent region of the notch can be set to the rotor region,
-so that the notch is set as a sub-region.
-Motor-CAD uses implicit subtractions,
-so that the notch subtraction is handled automatically.
-The notch appears as a sub-region of the rotor
-in the Geometry -> Editor tab in Motor-CAD.
+The parent region of the notch can be set to the rotor region, so that the notch is set as a
+sub-region. Motor-CAD uses implicit subtractions, so that the notch subtraction is handled
+automatically. The notch appears as a sub-region of the rotor in tree shown in the
```

Comment:
```suggestion
The parent region of the notch can be set to the rotor region so that the notch is set as a
sub-region. Motor-CAD uses implicit subtractions so that the notch subtraction is handled
automatically. The notch appears as a sub-region of the rotor in tree shown in the
```